diff --git a/CMakeLists.txt b/CMakeLists.txt
index 378e363..f817f42 100755
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -51,10 +51,9 @@ set( STATIC_BOOST        ON CACHE BOOL ${STATIC_BOOST_DESC} )
 
 if( INCLUDE_PYTHON )
   set( USE_PYTHON_VERSIONS_DESC "Specifies which version of Python to build Malmo with Python bindings" )
-  set( MACOS_USE_PYTHON_MODULE_DESC "Specifies which Python module to build Malmo on Apple MacOS" )
+  set( USE_PYTHON_MODULE_DESC "Specifies which Python module to build Malmo" )
   set( USE_PYTHON_VERSIONS 3.6 CACHE STRING ${USE_PYTHON_VERSIONS_DESC} )
-  # Boost has switched to using a 2 digit naming convention for python on MacOS.
-  set( MACOS_USE_PYTHON_MODULE "python36" CACHE STRING ${MACOS_USE_PYTHON_MODULE_DESC} )
+  set( USE_PYTHON_MODULE "python3" CACHE STRING ${USE_PYTHON_MODULE_DESC} )
 endif()
 
 set( WARNINGS_AS_ERRORS OFF )
@@ -106,16 +105,13 @@ SET(Boost_ADDITIONAL_VERSIONS "1.67" "1.67.0")
 
 SET(BOOST_PYTHON_MODULE_NAME "python")
 if ( USE_PYTHON_VERSIONS VERSION_GREATER 3 )
-    if ( APPLE )
-        SET( BOOST_PYTHON_MODULE_NAME ${MACOS_USE_PYTHON_MODULE} )
-    else()
-        SET( BOOST_PYTHON_MODULE_NAME "python3" )
-    endif()
+    SET( BOOST_PYTHON_MODULE_NAME ${USE_PYTHON_MODULE} )
 endif()
 
 if( WIN32 )
   SET(Boost_USE_STATIC_LIBS ON)
-  find_package( Boost COMPONENTS chrono date_time filesystem iostreams program_options ${BOOST_PYTHON_MODULE_NAME} regex system thread zlib REQUIRED )
+  find_package( Boost COMPONENTS chrono date_time filesystem iostreams program_options ${BOOST_PYTHON_MODULE_NAME} regex system thread REQUIRED)
+  add_definitions(-DBOOST_ALL_NO_LIB=1)
 else()
   find_package( Boost COMPONENTS chrono date_time filesystem iostreams program_options ${BOOST_PYTHON_MODULE_NAME} regex system thread REQUIRED )
   set(CMAKE_THREAD_PREFER_PTHREAD TRUE)
diff --git a/scripts/python-wheel/windows_wheel.bat b/scripts/python-wheel/windows_wheel.bat
index f66a8da..b643538 100644
--- a/scripts/python-wheel/windows_wheel.bat
+++ b/scripts/python-wheel/windows_wheel.bat
@@ -1,4 +1,4 @@
-pip3 install twine wheel
+REM pip3 install twine wheel
 copy ..\..\build\install\Python_Examples\MalmoPython.lib package\malmo
 copy ..\..\build\install\Python_Examples\MalmoPython.pyd package\malmo
 copy ..\..\build\install\Python_Examples\malmoutils.py package\malmo
@@ -8,4 +8,4 @@ copy ..\..\Minecraft\launch_minecraft_in_background.py package\malmo
 cd package
 python setup.py bdist_wheel
 del package\malmo\MalmoPython.lib package\malmo\MalmoPython.pyd
-twine upload dist/*
+REM twine upload dist/*
